### PR TITLE
增加对商汤科技商汤商量系列模型的支持（已经和MindSearch项目一起测试OK）   Add support for SenseTime's SenseNova series LLM  (Has been tested OK with MindSearch project)

### PR DESCRIPTION
**Pull request by winer632**
增加对商汤科技商汤商量系列模型的支持（已经和MindSearch项目一起测试OK）  
Add support for SenseTime's SenseNova series LLM  (Has been tested OK with MindSearch project)

https://github.com/winer632/MindSearch/releases/tag/Adapt_To_SenseNova_LLM

### CODE REVIEW

File: lagent/version.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-__version__ = '0.2.3'
+__version__ = '0.2.1'
```
Comment (Harold-lkk) [inline lagent/version.py:2]:
```suggestion
__version__ = '0.2.3'
```

File: setup.py [hunk 2]
```diff
@@ -89,7 +89,7 @@ def gen_packages_items():
         long_description = file.read()
 
     setup(
-        name='lagent',
+        name='xlagent',
```
Comment (Harold-lkk) [inline setup.py:92]:
```suggestion
        name='lagent',
```

**Comment by winer632** [comment]
@Harold-lkk 已经更新，可以merge了

**Comment by winer632** [comment]
@liujiangning30

**Review by Harold-lkk — APPROVED** [review]

**Pull request merged**